### DISCUSSION TITLE
Question about AWS: instance requirements

### DISCUSSION THREAD

**Discussion by 10N10N** [broadinstitute/seqr#3541] [General]
Hi, 
I am from Baylor College of Medicine, Human Genome Sequencing Center. My team is trying to install seqr on AWS as a pilot project. I am trying to find out some information on AWS Instance requirements (vCPUs and RAM) to get a better idea. I apologize if this question has been asked before. 
I would greatly appreciate any help/information.
thanks.

**Comment by hanars** [comment]
AWS is not well supported at this time, so our team does not have any specific advice for this. In general, there are some minimum machine requirements listed in the README, so you should use those at a minimum. The amount of compute you need will also vary widely based on the amount of data you plan to load - a few exome samples have vastly different needs than thousands of genomes.

**Reply by 10N10N** [reply]
Thank you for the quick response @hanars
